site/gatsby-site/src/templates/post.js

```diff
@@ -14,10 +14,14 @@ import { useLayoutContext } from 'contexts/LayoutContext';
 
 export default function Post(props) {
   const {
-    data: { mdx },
+    data: { mdx: localMdx, enMdx },
     children,
   } = props;
 
+  let mdx = localMdx;
+
+  if (!mdx) mdx = enMdx;
```

Comment:
I personally prefer to write a new line on these `if` clauses. To avoid possible merge conflicts and keep consistency.
If you are OK, please change all PR files accordingly.

E.g: 
```
if (!mdx) {
  mdx = enMdx;
}
```